FueledUtils/KeyboardInsetHelper.swift

```diff
@@ -24,6 +31,12 @@ open class KeyboardInsetHelper: NSObject {
 			name: NSNotification.Name.UIKeyboardWillShow,
 			object: nil
 		)
+		nc.addObserver(
+			self,
+			selector: #selector(handleKeyboardNotification(_:)),
+			name: NSNotification.Name.UIKeyboardWillChangeFrame,
```

Comment:
Good addition. I actually had to add this once to a custom subclass of the helper.

Reply:
@stephanecopin I'm a little concerned it may result in event duplication and glitchy animations as a result, so we might want to add skipRepeats to the exposed property. Can you do some debugging in client your code to see if this is the case?

Reply:
@leontiy I'll test it out in the current project I'm working on, see that holds up

Reply:
@stephanecopin did you have a chance to address this?

Reply:
@leontiy I'll try to look into it with TE2, see what it changes